docs/operating-scylla/nodetool-commands/restore.rst

```diff
@@ -4,9 +4,16 @@ Nodetool restore
 
 **restore** - Load SSTables from a designated bucket in object store into a specified keyspace or table
 
-Note that status of restore can be checked for ``user_task_ttl`` seconds after the operation is done.
-You can set the ttl using :doc:`nodetool tasks user-ttl </operating-scylla/nodetool-commands/tasks/user-ttl>`.
-If ``--nowait`` flag is not set, the command relies on ``user_task_ttl`` internally.
+The status of a restore operation is retained for a period defined by ``user_task_ttl``,
+allowing you to query the status even after the operation completes.
+You can configure the TTL duration using the :doc:`nodetool tasks user-ttl </operating-scylla/nodetool-commands/tasks/user-ttl>` command.
+
+To run the command asynchronously, use the ``--nowait`` flag. The command will then print a task ID.
+You can use this ID with :doc:`nodetool tasks </operating-scylla/nodetool-commands/tasks/index>` command to track the progress of the restore operation or
+to cancel it.
+
+To run the command synchronously, run it without the ``--nowait`` flag. In this case,
+the command relies on ``user_task_ttl`` internally.
```

Comment:
To me it is not clear what would happen, what is exactly synchronous. 
To me it is not clear what information I gain by knowing the command will use `user_task_ttl` internally. 
I think this sentence should be rephrased.

Reply:
@Deexie hi Aleksandra, could you help on this?

Reply:
Regardless of the presence of `--nowait` flag, `nodetool restore` requests asynchronous API and gets the task_id of the operation. 

If `--nowait` is set, the task_id is returned. The task_id can be used to check the status of the restore or wait for it using task manager. After the operation finishes, the task and its data will be available for `user_task_ttl`.

If `--nowait` isn't set, the task_id is used by nodetool to wait for the operation and the status of the finished restore is returned. However, if the operation is finished immediately and the value of `user_task_ttl` is small, suppose 0 (which means that the task won't be visible after it's done), we may end up with the restore task being unregistered before nodetool manages to check its status.